### ISSUE TITLE
Snippets in `snippets.cson` are not loaded

### ISSUE THREAD

**Issue by timnew** [atom/snippets#80]
I'm using `Atom` `0.141.0` on Mac. 
And I found all the snippets in `snippets.cson` doesn't work, the `Snippets` plugin is `0.141.0`
Press `Tab` the snippet is not expanded.
And press `Alt+Shift+S` to open `snippets:available` list, the snippets in `snippets.cson` are not listed.
Other predefined snippets works well. 

So I assume it seems that the snippets in `snippets.cson` are not loaded.

**Comment by Srogozins** [comment]
I keep having this problem even after saving the file (the error just pops up again). I doubt it's a formatting error since I'm using the code samples from atom's snippets guide.

**Comment by Srogozins** [comment]
Seems to have magically fixed itself for now. The only thing I did was delete everything from the snippets.cson file and save it.

**Comment by izuzak** [comment]
> I keep having this problem even after saving the file (the error just pops up again).

@Srogozins which error did you see exactly? If you run into this again, can you please provide as many details as possible?

Since atom/atom#1867 was closed and we didn't get more information from @timnew about the problems he was seeing, I'm going to close this. If anyone can still reproduce this -- please let us know and provide your snippets.cson file. :v:

**Comment by admmasters** [comment]
I can reproduce this easily. Snippets are virtually worthless to me in Atom. Incredibly buggy. I'm happy to provide the .cson file I'm using.

**Comment by admmasters** [comment]
Combinations of deleting then re-saving seem to work.

**Comment by admmasters** [comment]
Hmmm - are you allowed more than one snippet in snippets.cson? I have:
```
'.source.js':
  'React Native Import':
    'prefix': 'rni'
    'body': """
    import React from 'react-native';
    """
'.source.js':
  'React Native Import':
    'prefix': 'irn'
    'body': """
    import React from 'react-native';
    """

```
Wondering if Atom only allows one snippet per file?

**Comment by admmasters** [comment]
This helped: http://stackoverflow.com/questions/26615818/atom-editor-multiple-snippets

**Comment by admmasters** [comment]
If anyone wants to know the solution for me was:
```
'.source.js':
  'React Native Import 1':
    'prefix': 'rni'
    'body': """
    import React from 'react-native';
    """
  'React Native Import 2':
    'prefix': 'rnx'
    'body': """
    import React from 'react-native';
    """
```

**Comment by danorr** [comment]
It would appear that any given scope can only be declared once. I had the same issue and after reading the blurb at the top of the snippets.cson, removed all but the first declaration of the scope and all snippets became available.

**Comment by jhanstra** [comment]
I made an npm package called [snipster](https://www.npmjs.com/package/snipster) that drastically simplifies the process of writing and managing snippets - not only for Atom but for VSCode and Sublime (and more to come) as well. You might find it easier than hand-writing Atom's CSON snippet file. I wrote a [blog post](http://www.jaredhanstra.com/blog/2017-05-09-snipster/) about the problems I was trying to solve and the solution I came up with. Hope you find it helpful!

@danorr @admmasters @yihangho @timnew
